### PR TITLE
8.0 Use try/except when importing external libs

### PR DESCRIPTION
**Pull request by alexis-via**
Remove self.ensure_one() that has nothing to do in an api.model method

Fix bug #7

### CODE REVIEW

File: account_invoice_import_invoice2data/wizard/account_invoice_import.py [hunk 1]
```diff
@@ -6,14 +6,17 @@
...
-
+import logging
 logger = logging.getLogger(__name__)
 
+try:
+    from invoice2data.main import extract_data
+    from invoice2data.template import read_templates
+    from invoice2data.main import logger as loggeri2data
+except ImportError:
+    logger.warning('Cannot import invoice2data')
```
Comment (moylop260) [inline account_invoice_import_invoice2data/wizard/account_invoice_import.py]: s/warning/debug/g
Like as our example from guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#importerror

File: account_invoice_zugferd/models/account_invoice.py [hunk 2]
```diff
@@ -8,15 +8,19 @@
...
-# from pprint import pprint
-
 logger = logging.getLogger(__name__)
+
+try:
+    from PyPDF2 import PdfFileWriter, PdfFileReader
+    from PyPDF2.generic import DictionaryObject, DecodedStreamObject,\
+        NameObject, createStringObject, ArrayObject
+except ImportError:
+    logger.warning('Cannot import PyPDF2')
```
Comment (moylop260) [inline account_invoice_zugferd/models/account_invoice.py]: Same case

**Review by moylop260 — CHANGES_REQUESTED** [review]
Thanks for your PR

Could you create a [requirements.txt](https://github.com/OCA/maintainer-quality-tools/blob/master/requirements.txt) in the root of the project (MQT automatically use it to install dependencies) and remove `pip install` from .travis.yml file?

**Review by pedrobaeza — APPROVED** [review]
Thanks for the followup. I fast-track to fix the branch.

**Pull request merged**